### PR TITLE
Fix all project coding warnings in `examples` folder

### PR DESCRIPTION
**Pull request by hanton**
Fix all project coding warnings in `examples` folder

### CODE REVIEW

**Comment by pinterest-ios** [comment]
🚫 CI failed with [log](https://gist.githubusercontent.com/pinterest-ios/7711c598c6c60cccca59bffceaab53b9/raw/72421364a7f931c6224a59934234c281b5c2fdff/log.txt)

**Comment by hanton** [comment]
Could you trigger the CI again? 

Because it seems the CI failing is not related to the changed code, according to the error message ^_^

```
** BUILD FAILED **


The following build commands failed:
	CompileAssetCatalog /Users/buildkite/ASDKDerivedData/Build/Products/Debug-iphonesimulator/Sample.app /Users/Shared/buildkite/builds/iosf-garrett-vuqKsO4.dyn.pinadmin.com-1/pinterest/texture/examples/CustomCollectionView/Sample/Images.xcassets
(1 failure)
```

File: examples/VerticalWithinHorizontalScrolling/Sample/GradientTableNode.mm [hunk 1]
```diff
@@ -58,7 +58,8 @@ - (ASCellNode *)tableNode:(ASTableNode *)tableNode nodeForRowAtIndexPath:(NSInde
 {
```
Comment (nguyenhuy) [inline examples/VerticalWithinHorizontalScrolling/Sample/GradientTableNode.mm]: We should migrate to the `-nodeBlock` data source method as well. Not a blocker for this PR though.

File: examples/VerticalWithinHorizontalScrolling/Sample/GradientTableNode.mm [hunk 2]
```diff
@@ -58,7 +58,8 @@ - (ASCellNode *)tableNode:(ASTableNode *)tableNode nodeForRowAtIndexPath:(NSInde
...
   RandomCoreGraphicsNode *elementNode = [[RandomCoreGraphicsNode alloc] init];
   elementNode.style.preferredSize = _elementSize;
-  elementNode.indexPath = [NSIndexPath indexPathForRow:indexPath.row inSection:_pageNumber];
+  NSString *content = [[NSIndexPath indexPathForRow:indexPath.row inSection:_pageNumber] description];
+  [elementNode setContent:content];
```
Comment (nguyenhuy) [inline examples/VerticalWithinHorizontalScrolling/Sample/GradientTableNode.mm]: If the content is set only once, maybe pass it in as a param of the initializer and remove `-setContent:` altogether?

**Comment by nguyenhuy** [comment]
You got a green light from CI btw.

File: examples/VerticalWithinHorizontalScrolling/Sample/GradientTableNode.mm (hunk 1 above)
Comment (hanton) [reply examples/VerticalWithinHorizontalScrolling/Sample/GradientTableNode.mm]: I find there're a few other example projects which also need to migrate to the `-nodeBlock` data source method, will create a new PR for fixing this issue altogether :)

File: examples/VerticalWithinHorizontalScrolling/Sample/GradientTableNode.mm (hunk 2 above)
Comment (hanton) [reply examples/VerticalWithinHorizontalScrolling/Sample/GradientTableNode.mm]: fixed.

File: examples/VerticalWithinHorizontalScrolling/Sample/GradientTableNode.mm (hunk 1 above)
Comment (hanton) [reply examples/VerticalWithinHorizontalScrolling/Sample/GradientTableNode.mm]: hi @nguyenhuy , the PR is ready at #1596

**Pull request merged**